### DISCUSSION TITLE
貌似对神策的send_type的类型是beacon的数据接收有问题

### DISCUSSION THREAD

**Discussion by jiajiano654321** [white-shiro-bai/ghost_sa#13] [Q&A]
收不到，不知道是什么原因

**Comment by white-shiro-bai** [comment]
beacon确实有这个问题。收到了，解不出，还在努力想办法解决。

https://manual.sensorsdata.cn/sa/latest/sdk-api-web-7549300.html
webjs建议还是用image方式上报，比较保险，神策官方也不是很推荐beacon的方式。

**Comment by white-shiro-bai** [comment]
#27 已创建问题
